diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter1/IdentifierNaming.kt

```diff
@@ -139,6 +145,34 @@ class IdentifierNaming(configRules: List<RulesConfig>) : DiktatRule(
         return false
     }
 
+    /**
+     * method checks that identifier is correct backing field
+     */
+    private fun ASTNode.isCorrectBackingField(variableName: ASTNode): Boolean {
+        val propertyNodes = this.treeParent.getAllChildrenWithType(KtNodeTypes.PROPERTY)
+        // check that backing field name is correct
+        if (variableName.text.commonPrefixWith("_").length == 1 && variableName.text.drop(1).isLowerCamelCase()) {
```

Comment:
Why do you do this trick with `commonPrefixWith.length == 1` -> it should be `startsWith('_')`, no?

Reply:
guess, that `startsWith` not take into account cases like `____bla`, but `.len == 1` does

Reply:
Changed to `startsWith("_")`